### PR TITLE
Feature/docstring -> develop

### PR DESCRIPTION
**Pull request by Reagen**
Updated all docstrings to follow the same format as described in Issue #64.

### CODE REVIEW

File: analysis/deep2_r23_o32/binning/double_binning_grid.py [hunk 1]
```diff
@@ -12,21 +12,24 @@
...
-      One_dimensional binning for R23 followed by each bin being split in
-      half one with high O32 and one with low O32
+    One_dimensional binning for R23 followed by each bin being split in
+    half one with high O32 and one with low O32
 
     Followed by plotting the spectra in their R23-O32 bins for visual
 
     :param fitspath: str. Path where files are retrieved and saved to
     :param pdf_pages: str. Name of outputted pdf file
+                        fitspath + name_dict['gridpdf_suffix']
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/double_binning_grid.py:22]:
```suggestion
                      fitspath + name_dict['gridpdf_suffix']
```

File: analysis/deep2_r23_o32/binning/fixed_grid_analysis.py [hunk 2]
```diff
@@ -13,16 +13,22 @@
...
+    This file holds the function to bin data in a bin of fixed size entered
+    as an input. Not used in current analysis
 
     :param fitspath: str. Path where files are retrieved and saved to
-    :param pdf_pages: PdfPages object
-    :param npz_outfile: name of the npz file produced by the function
-    :param R23_bin: set size of the bin in the R23 direction
-    :param O32_bin: set size of the bin in the O32 direction
+    :param pdf_pages: str. PdfPages object
+                        fitspath + name_dict['gridpdf_suffix']
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/fixed_grid_analysis.py:21]:
```suggestion
    :param pdf_pages: str. PdfPages object
                      fitspath + name_dict['gridpdf_suffix']
```

File: analysis/deep2_r23_o32/binning/n_bins_grid_analysis.py [hunk 3]
```diff
@@ -14,16 +14,17 @@
...
+    This file holds the function to bin data adaptively based on the entered
+    number of galaxies for each bin.
+    First, it bins in the R23 direction and then in the O32 direction
 
-      This is used in current analysis.
+    This is used in current analysis.
 
     :param fitspath: str. Absolute path for working directory
     :param pdf_pages: str. name of outputted pdf file
+                        fitspath + name_dict['gridpdf_suffix']
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/n_bins_grid_analysis.py:25]:
```suggestion
    :param pdf_pages: str. name of outputted pdf file
                      fitspath + name_dict['gridpdf_suffix']
```

File: analysis/deep2_r23_o32/binning/single_grid_o32.py [hunk 4]
```diff
@@ -8,16 +8,19 @@
...
-    Purpose:
-      This file holds the function to bin data for one dimensional binning along
-      log(O32) Not used in current analysis
+    This file holds the function to bin data for one dimensional binning along
+    log(O32) Not used in current analysis
 
-    :param pdf_pages: name of outputted pdf file
-    :param npz_outfile: name of the npz file produced by the function
+    :param pdf_pages: str. name of outputted pdf file
+                        fitspath + name_dict['gridpdf_suffix']
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/single_grid_o32.py:15]:
```suggestion
    :param pdf_pages: str. name of outputted pdf file
                      fitspath + name_dict['gridpdf_suffix']
```

File: analysis/deep2_r23_o32/binning/single_grid_r23.py [hunk 5]
```diff
@@ -8,17 +8,20 @@
...
-      Note: Not used in current analysis
-
-    :param pdf_pages: name of outputted pdf file
-    :param npz_outfile: name of the npz file produced by the function
+    This file holds the function to bin data for one dimensional binning
+    along R23.
+    Note: Not used in current analysis
+
+    :param pdf_pages: str. name of outputted pdf file
+                        fitspath + name_dict['gridpdf_suffix']
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/single_grid_r23.py:16]:
```suggestion
    :param pdf_pages: str. name of outputted pdf file
                      fitspath + name_dict['gridpdf_suffix']
```

File: analysis/deep2_r23_o32/calibration_plots.py [hunk 6]
```diff
@@ -191,6 +193,15 @@ def individual_gpc(individual_ascii, validation_table, log=None):
...
     the binned detections.
+
+    :param individual_ascii: str. location of table with data
+    :param validation_table: str. location of validaition table
+    :param log LogClass or logging object
+
+    PDF Files:
+        fitspath + GPC{suffix}.pdf
+        fitspath + GPC{suffix}.diff.pdf
+    No returns
```
Comment (astrochun) [inline analysis/deep2_r23_o32/calibration_plots.py:204]:
```suggestion

    No returns
```

File: analysis/deep2_r23_o32/calibration_plots.py [hunk 7]
```diff
@@ -191,6 +193,15 @@ def individual_gpc(individual_ascii, validation_table, log=None):
     This function is currently repetitive of the function above for the individual detection cases.
     However, I am going to keep it here in the case that we want to plot the individual detections against
     the binned detections.
+
+    :param individual_ascii: str. location of table with data
+    :param validation_table: str. location of validaition table
+    :param log LogClass or logging object
```
Comment (astrochun) [inline analysis/deep2_r23_o32/calibration_plots.py:199]:
```suggestion
    :param log: LogClass or logging object
```

File: analysis/deep2_r23_o32/calibration_plots.py [hunk 8]
```diff
@@ -191,6 +193,15 @@ def individual_gpc(individual_ascii, validation_table, log=None):
     This function is currently repetitive of the function above for the individual detection cases.
     However, I am going to keep it here in the case that we want to plot the individual detections against
     the binned detections.
```
Comment (astrochun) [inline analysis/deep2_r23_o32/calibration_plots.py:195]: Consider limited it to 79 characters wide by wrapping the text

File: analysis/deep2_r23_o32/log_commons.py [hunk 9]
```diff
@@ -15,18 +15,18 @@
...
     To use:
     log = LogClass(log_dir, logfile).get_logger()
 
-    Parameters:
-      log_dir: Relative path for exported logfile directory
-      logfile: Filename for exported log file
+    :param log_dir: Relative path for exported logfile directory
+    :param logfile: Filename for exported log file
+
+    :return log
```
Comment (astrochun) [inline analysis/deep2_r23_o32/log_commons.py:29]:
```suggestion
    :return log: LogClass object
```

**Review by astrochun — COMMENTED** [review]
Here are a few suggestions. It's probably easier for me to do this within my IDE, so I'll switch over and commit changes.

File: analysis/deep2_r23_o32/calibration_plots.py (hunk 8 above)
Comment (Reagen) [reply analysis/deep2_r23_o32/calibration_plots.py]: I thought PyCharm was set to 79, but it turns out it was set to 120. Went back through code and fixed this. commit 359a259

File: analysis/deep2_r23_o32/calibration_plots.py [hunk 10]
```diff
@@ -209,7 +228,8 @@ def individual_gpc(individual_ascii, validation_table, log=None):
 
     log.info("starting ...")
 
-    pea_out_pdf_ind = '/Users/reagenleimbach/Desktop/Zcalbase_gal/R23O32_Manual_0417/jiang_plot_individual.pdf'
+    pea_out_pdf_ind = '/Users/reagenleimbach/Desktop/Zcalbase_gal/' \
+                       'R23O32_Manual_0417/jiang_plot_individual.pdf'
```
Comment (astrochun) [inline analysis/deep2_r23_o32/calibration_plots.py:232]: Why is this still hardcoded to non-Drive path?

**Review by astrochun — COMMENTED** [review]
This is good for now. Let's create a different issue for PEP8.

File: analysis/deep2_r23_o32/calibration_plots.py (hunk 10 above)
Comment (Reagen) [reply analysis/deep2_r23_o32/calibration_plots.py:232]: Fixed it. Sorry about that.

**Review by astrochun — APPROVED** [review]
These changes look good. Let's create a separate issue for any PEP8 formatting issue.

**Pull request merged**